_includes/schemas/shipments_response.json

```diff
@@ -2,38 +2,34 @@
   "$schema": "http://json-schema.org/draft-04/schema#",
   "type": "object",
   "properties": {
-    "id": {
+    "carrier": {
       "type": "string",
-      "description": "the shipment id that can be used for requesting info about a shipment or tracking it"
+      "enum": ["dhl", "ups", "dpd", "hermes", "gls"],
```

Comment:
I think there are some carriers missing here. In a different PR you used this collection:

`["dhl", "dhl_express", "dpag", "ups", "dpd", "hermes", "iloxx", "gls", "tnt", "fedex", "liefery"]`